benchmark/v5.1-performance-benchmarking-with-tpch.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: TiDB TPC-H 100 性能对比测试报告 - v5.0 MPP 模式对比 Greenplum 6.15.0 以及 Apache Spark 3.1.1
+title: TiDB TPC-H 100 性能对比测试报告 - v5.1 MPP 模式对比 Greenplum 6.15.0 以及 Apache Spark 3.1.1
 ---
 
-# TiDB TPC-H 性能对比测试报告 - v5.0 MPP 模式对比 Greenplum 6.15.0 以及 Apache Spark 3.1.1
+# TiDB TPC-H 性能对比测试报告 - v5.1 MPP 模式对比 Greenplum 6.15.0 以及 Apache Spark 3.1.1
 
 ## 测试概况
 
-本次测试对比了 TiDB v5.0 MPP 模式下和主流分析引擎例如 Greenplum 和 Apache Spark 最新版在 TPC-H 100 下的性能表现。结果显示，TiDB v5.0 MPP 模式下相对这些方案有 2-3 倍的性能提升。
+本次测试对比了 TiDB v5.1 MPP 模式下和主流分析引擎例如 Greenplum 和 Apache Spark 最新版在 TPC-H 100 下的性能表现。结果显示，TiDB v5.1 MPP 模式下相对这些方案有 2-3 倍的性能提升。
 
 TiDB v5.0 中新引入了 [TiFlash](/tiflash/tiflash-overview.md) 组件的 MPP 模式，这大幅增强了 TiDB HTAP 形态。本文的测试对象如下：
```

Comment:
Do we need to update this sentence for v5.1 to show the stability improvement of 5.1 MPP? @ilovesoup

Reply:
There is no update in TPC-H directly in MPP in 5.1. I think update like this is ok.